src/DotVVM.Samples.Common/DotvvmStartup.cs

```diff
@@ -113,7 +113,6 @@ private static void AddControls(DotvvmConfiguration config)
             config.Markup.AddCodeControls("cc", typeof(Controls.PromptButton));
             config.Markup.AddMarkupControl("IdGeneration", "Control", "Views/FeatureSamples/IdGeneration/IdGeneration_control.dotcontrol");
             config.Markup.AddMarkupControl("FileUploadInRepeater", "FileUploadWrapper", "Views/ComplexSamples/FileUploadInRepeater/FileUploadWrapper.dotcontrol");
-            config.Markup.AddMarkupControl("sample", "PasswordStrengthControl", "Views/FeatureSamples/ClientExtenders/PasswordStrengthControl.dotcontrol");
```

Comment:
You are right that this route is wrong, but it would be nicer to replace it with the correct one instead of removing it :) The correct file is at `Views/FeatureSamples/ClientExtenders/PasswordStrength.dothtml`

Reply:
This route is no longer needed. The `Views/FeatureSamples/ClientExtenders/PasswordStrength.dothtml` is automatically resolved by route strategy.